ui/selectors/selectors.js

```diff
@@ -113,8 +113,8 @@ import {
   SURVEY_START_TIME,
 } from '../helpers/constants/survey';
 import { PRIVACY_POLICY_DATE } from '../helpers/constants/privacy-policy';
-import { SUPPORTED_CHAIN_IDS } from '../../app/scripts/lib/ppom/ppom-middleware';
 import { ENVIRONMENT_TYPE_POPUP } from '../../shared/constants/app';
+import { PPOM_SUPPORTED_CHAIN_IDS } from '../../app/scripts/lib/ppom/constants';
```

Comment:
It may be better to place above constants in shared folder. This helps ensure that none of `/app` code is bundled in client side build.

Reply:
Done.